### PR TITLE
Merging m2 master into master

### PR DESCRIPTION
**Pull request by ivannaranjo**
Now that we have released v1.1.7.0 of the extension publicly it is time to move on to the next release. This PR brings the work that we have been done on the `m2_master` branch back into the `master` branch as it is the work slated for release next. The contents of the PR are unchanged from `m2_master` so hopefully not much review is needed, but there were some merge conflicts that were resolved by hand. The main file to check is Resources.resx, to ensure that all the strings are well formed. 

After this PR is merged the `m2_master` branch will be retired and will open a new branch, m3_master, to work on the next wave.

### CODE REVIEW

File: GoogleCloudExtension/GoogleCloudExtension/CloudExplorerSources/Gae/VersionViewModel.cs [hunk 1]
```diff
@@ -202,6 +210,7 @@ private void OnOpenVersion()
                 _owner.InvalidateService(_service.Id);
 
                 EventsReporterWrapper.ReportEvent(GaeVersionDeletedEvent.Create(CommandStatus.Success));
+                _owner.InvalidateService(_service.Id);
```
Comment (Deren-Liao) [inline GoogleCloudExtension/GoogleCloudExtension/CloudExplorerSources/Gae/VersionViewModel.cs]: Remove this.

Comment (Deren-Liao) [inline GoogleCloudExtension/GoogleCloudExtension/CloudExplorerSources/Gae/VersionViewModel.cs]: Is this duplicated ? An error from merging somewhere or is this intended to do InvalidateService multiple times?

**Comment by iantalarico** [comment]
Sorry for the delay, I will TAL at this today

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx [hunk 2]
```diff
@@ -1167,34 +1171,66 @@
     <value>_Allocations:</value>
     <comment>Label shown in the split traffic window to indicate the list of allocations.</comment>
   </data>
+  <data name="LogViewerCollapseAllTip" xml:space="preserve">
+    <value>Click to hide all log items detail.</value>
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: This is out of the scope of this PR but this and the below comment are a little hard to parse

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx [hunk 3]
```diff
@@ -1203,10 +1239,122 @@
...
   <data name="AnalyticsOptionsOptInDisplayName" xml:space="preserve">
     <value>Report Usage Statistics Enabled</value>
     <comment>The display name of the opt-in for the analytics options page.</comment>
   </data>
+  <data name="LogViewerTitleBarProjectIdLabel" xml:space="preserve">
+    <value>Project</value>
+    <comment>Label for project id label.</comment>
+  </data>
+  <data name="LogViewerRequestCancellingMessage" xml:space="preserve">
+    <value>Cancelling . . .</value>
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Again probably out of scope but is the  ". . ." normal?  I think we may need to review some of these log messages

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx [hunk 4]
```diff
@@ -1303,6 +1451,77 @@
...
+    <comment>The warning message when a log entry does not contain valid version information.</comment>
+  </data>
+  <data name="LogsViewerVersionMismatchPrompt" xml:space="preserve">
+    <value>Version missmatch. 
+The current project {0} version is {1},
+Open the project of version {2} to properly locate the logging writter location.</value>
+    <comment>Prompt when the opened project version differs than the version information of the log entry.</comment>
+  </data>
+  <data name="LogsViewerVersionMissmatchAskIgnoreMessage" xml:space="preserve">
+    <value>"Do you want to continue anyway?"</value>
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Should this be quoted?

Comment (Deren-Liao) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: This should not hold this merge though.
I'll see if this looks wrong.    I opened #502 for this.

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx (hunk 3 above)
Comment (Deren-Liao) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Maybe not normal.  It's  a easy solution to indicate the cancelling is in progress.

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx (hunk 2 above)
Comment (Deren-Liao) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Can you elaborate what's not clear?   This PR is about merging.   Please feel free to open issue to me for things that does not look right.  It is not a "merge" error, making changes in this "merge" PR does not make sense.

**Comment by iantalarico** [comment]
@Deren-Liao I don't think any of this should hold the merge (as noted they are out of scope) given they were the error messages from before.  

@ivannaranjo  Feel free to merge if nothing else need to be reviewed.  Sorry if my comments caused confusion.

File: GoogleCloudExtension/GoogleCloudExtension/CloudExplorerSources/Gae/VersionViewModel.cs (hunk 1 above)
Comment (ivannaranjo) [inline GoogleCloudExtension/GoogleCloudExtension/CloudExplorerSources/Gae/VersionViewModel.cs]: Thanks for picking this up, this is indeed a bad merge. Will update and merge back.

**Comment by ivannaranjo** [comment]
Thank you @Deren-Liao and @iantalarico for reviewing the changes.

And yes we will go through text review before the final release.
